### ISSUE TITLE
Error on server startup with Docker image

### ISSUE THREAD

**Issue by Capricom** [OpenFactorioServerManager/factorio-server-manager#195]
I'm trying to create a docker container from the ui-rework branch which has now been merged into develop, and I get this error on container startup:

`Admin username is 'admin',
Admin password is 'factorio',
IMPORTANT! Please create new user and delete default admin user ASAP.,
Factorio rcon password is 'crR3f733aLnYg5pKyOq3bzMN',
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current,
                                 Dload  Upload   Total   Spent    Left  Speed,

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   445  100   445    0     0    568      0 --:--:-- --:--:-- --:--:--   568,

 11 48.0M   11 5615k    0     0  5999k      0  0:00:08 --:--:--  0:00:08 5999k
100 48.0M  100 48.0M    0     0  36.5M      0  0:00:01  0:00:01 --:--:--  112M,
/opt/fsm/mod_packs,
2020/12/09 12:03:17 no mods dir found ... creating one ...,
2020/12/09 12:03:17 no ModPackDir found ... creating one ...,
2020/12/09 12:03:17 Server settings at config/server-settings.json not found, copying example server settings.,
2020/12/09 12:03:17 failed to open example server settings: open data/server-settings.example.json: no such file or directory,
2020/12/09 12:03:17 Error occurred during Server initializaion: open data/server-settings.example.json: no such file or directory,`

This happens with both the docker image built with docker/build.sh and with the image from dockerhub ofsm/ofsm:develop.

Is this a known problem or am I doing something wrong?

Thanks

**Comment by cheetahx7x** [comment]
This is probably due to the fact that in the entrypoint.sh the fsm binary is getting executed with some long arguments which do not have a double dash in front. It should look like this:
./factorio-server-manager --conf /opt/fsm-data/conf.json --dir /opt/factorio --port 80
but it looks like this:
./factorio-server-manager -conf /opt/fsm-data/conf.json -dir /opt/factorio -port 80

I hope i could help.
:-)

**Comment by knoxfighter** [comment]
yep, that was it, i fixed it :)
Will be in the next release.

**Comment by knoxfighter** [comment]
This error is back again:
```
knox@knox-Lenovo-Y50-70:~/Desktop/temp/fsm-docker$ docker-compose up
Creating network "fsm-docker_default" with the default driver
Pulling factorio-server-manager (ofsm/ofsm:develop)...
develop: Pulling from ofsm/ofsm
df20fa9351a1: Already exists
05154a797e78: Already exists
8b035276a627: Pull complete
f5a49f57c4df: Pull complete
b64b420ec63c: Pull complete
09e7829a2180: Pull complete
Digest: sha256:c2ff63baa61b910effb9d8eccc78275624197abb4f7d6e9bd0ff2eb50e040edb
Status: Downloaded newer image for ofsm/ofsm:develop
Creating factorio-server-manager ... done
Attaching to factorio-server-manager
factorio-server-manager    | Admin username is 'admin'
factorio-server-manager    | Admin password is 'factorio'
factorio-server-manager    | IMPORTANT! Please create new user and delete default admin user ASAP.
factorio-server-manager    | Factorio rcon password is 'ePffPNktGfyZNs3lXV1cfbnC'
factorio-server-manager    |   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
factorio-server-manager    |                                  Dload  Upload   Total   Spent    Left  Speed
100   445  100   445    0     0    474      0 --:--:-- --:--:-- --:--:--   473
100 48.1M  100 48.1M    0     0  11.7M      0  0:00:04  0:00:04 --:--:-- 16.2M
factorio-server-manager    | /opt/fsm/mod_packs
factorio-server-manager    | 2020/12/27 20:12:50 no ModPackDir found ... creating one ...
factorio-server-manager    | 2020/12/27 20:12:50 Server settings at /opt/factorio/config/server-settings.json not found, copying example server settings.
factorio-server-manager    | 2020/12/27 20:12:50 Loaded Factorio settings from /opt/factorio/config/server-settings.json
factorio-server-manager    | 2020/12/27 20:12:50 Error occurred during Server initializaion: stat /opt/factorio/config/server-adminlist.json: no such file or directory
```

**Comment by knoxfighter** [comment]
Fixed again with 65b3b20
